### DISCUSSION TITLE
Request for help with static B1 shimming

### DISCUSSION THREAD

**Discussion by gxu32** [shimming-toolbox/shimming-toolbox#521] [Q&A] [answered]
Dear experts
I'm using the shimmingtoolbox to do B1 shimming. We have 8 transmit channels and the initial phase of default setting is 0 -45 -90 -135 -180 -225 -270 -315. 
I encountered two problems during the B1 shimming process. I would greatly appreciate it if you could provide me with some suggestions. 
1. Based on your tutorial, the B1 map should represent the field strength of the B1+. But we only have the B1 map of flip angle, I wonder if I need to convert the flip angle map to field strength?

2. The results in 'b1_shim_weights_hrd.txt' represent the magnitudes and phases of each channel. I'm curious if the resulting phases indicate the offset value that needs to be applied to the initial phase of the default setting(0 -45 -90 -135 -180 -225 -270 -315), or they are the absolute values that can be directly replace the initial phase of default setting.

Many Thanks!
Best,
gaoqiang

**Comment by po09i** [comment, marked as answer]
Hi @gxu32
1: The toolbox has been set up to accept the field strength so it is better to use that. However, some algorithms will still work. For example, the optimization for a particular value will work as long as the units match. I believe the CoV will also work but I have not tested it myself.
2: The values are directly the ones to apply. (i.e.: they are not an offset)
Best,
Alex

**Reply by gxu32** [reply]
Thank you(^_^)！
